packages/color-field/README.md

```diff
@@ -0,0 +1,107 @@
+## Description
+
+### Usage
+
+[![See it on NPM!](https://img.shields.io/npm/v/@spectrum-web-components/color-field?style=for-the-badge)](https://www.npmjs.com/package/@spectrum-web-components/color-field)
+[![How big is this package in your project?](https://img.shields.io/bundlephobia/minzip/@spectrum-web-components/color-field?style=for-the-badge)](https://bundlephobia.com/result?p=@spectrum-web-components/color-field)
+
+```
+yarn add @spectrum-web-components/color-field
+```
+
+Import the side effectful registration of `<sp-color-field>` via:
+
+```
+import '@spectrum-web-components/color-field/sp-color-field.js';
+```
+
+When looking to leverage the `ColorField` base class as a type and/or for extension purposes, do so via:
+
+```
+import { ColorField } from '@spectrum-web-components/color-field';
+```
+
+## Example
+
+### Default
+
+```html
+<sp-color-field></sp-color-field>
+```
+
+### devMode
+
+When devMode is true, the color handle will be rendered. This is useful for development and debugging purposes.
+
+```html
+<sp-color-field devMode="true"></sp-color-field>
+```
+
+### Read Only
+
+A readonly color field
+
+```html
+<sp-color-field readonly value="#ffff00"></sp-color-field>
+```
+
+### Quiet
+
+A readonly color field
+
+```html
+<sp-color-field quiet value="#ffff00"></sp-color-field>
+```
+
+### Wrong Input
+
+If the input value is not a valid color, the sp-color-field will not accept it. The input field will be cleared.
+
+```html
+<sp-color-field value="not a color"></sp-color-field>
+```
+
+### Right Input
+
+If the input value is a valid color, the sp-color-field will accept it and the color handle will be updated to reflect the new color.
```

Comment:
If the input value is a valid color, the `<sp-color-field>` will accept it and the color handle will be updated to reflect the new color.